cmd/util/ledger/migrations/staged_contracts_migration.go

```diff
@@ -239,7 +239,17 @@ func (m *StagedContractsMigration) collectAndRegisterStagedContractsFromPayloads
 	inter := mr.Interpreter
 	locationRange := interpreter.EmptyLocationRange
 
-	storageMap := mr.Storage.GetStorageMap(stagingAccountAddress, common.PathDomainStorage.Identifier(), false)
+	storageMap := mr.Storage.GetStorageMap(
+		stagingAccountAddress,
+		common.PathDomainStorage.Identifier(),
+		false,
+	)
+	if storageMap == nil {
+		m.log.Error().
+			Msgf("failed to get staged contracts from account %s", stagingAccount)
+		return nil
```

Comment:
Do we want the rest of the migrations to continue? If not maybe returning an error is a better option?

Reply:
Yeah, this was intentional: Though in a real run it's unlikely that the account is missing, it's still useful for development, when trying to run the migration on a subset of accounts.

Maybe it should error by default, but continue with a flag?

Reply:
TBH the way it is now is also fine; Don't want to make it overly complex. Like you've said, it's very unlikely this will happen in the real migration, so  just printing an error should be sufficient too for debugging.